### DISCUSSION TITLE
Plugin crashes

### DISCUSSION THREAD

**Discussion by monefin** [VirtusLab-Open-Source/strapi-plugin-comments#52] [Q&A] [answered]
Hi guys,
I am very new to Strapi (just started playing around today 🙂   and it looks great) and I was wondering how this Comments plugin works and wanted to have a play around with it as well as this plugin seems to look amazing by installing it in Strapi quick starter but it crashes with the following error every time I visit comment collection type `TypeError: Cannot read property 'pluginOptions' of undefined` . I think I have followed the instruction correctly. 
![Screen Shot 2021-04-28 at 8 50 54 pm](https://user-images.githubusercontent.com/1200216/116393759-6388b900-a865-11eb-8e92-5b0e57e07da7.png)
![Screen Shot 2021-04-28 at 9 01 27 pm](https://user-images.githubusercontent.com/1200216/116393773-6683a980-a865-11eb-9cdd-51f0f4f9a3f2.png)

I looked the error up and I could not find the answer so far and also tried quickly to add pluginOptions empty object as the comment object's attribute placeholder in the article.settings.json but the error seems to be persistent. so just wondering if you have encountered a similar issue when setting up this plugin. Thank you in advance

**Comment by cyp3rius** [comment, marked as answer]
Hi @monefin,
That's expected as plugin delivers own authoring UI. Check please the Plugins section on your screen -> Comments. You should use this. In the RBAC you can also exclude Comments & Reports collection types from viewing by other roles.

**Reply by monefin** [reply]
Oh okay I see now that's how that works. Thank you very much for your prompt response @cyp3rius and I do really appreciate it.
